onnxruntime/python/tools/quantization/quantize.py

```diff
@@ -213,6 +217,164 @@ def __init__(
         self.extra_options = extra_options or {}
 
 
+def get_int_qdq_config(
```

Comment:
>get_int_qdq_config [](http://example.com/codeflow?start=4&length=18)

is there non-int qdq config?

Reply:
Theres float8 quantization. This helper function was made with int only quantization in mind.